### PR TITLE
feat: add endpoint to send email to finance when user request a reprocess

### PR DESCRIPTION
**Pull request by ms-fcabrera**
add endpoint to send email to finance when user request a reprocess by transfer.
[DAT-1274](https://makingsense.atlassian.net/browse/DAT-1274)

### CODE REVIEW

File: Doppler.BillingUser/Controllers/BillingController.cs [hunk 1]
```diff
@@ -264,6 +264,20 @@ public async Task<IActionResult> UpdateCurrentPaymentMethod(string accountname,
                 return new BadRequestObjectResult(e.Message);
             }
         }
+        [Authorize(Policies.PROVISORY_USER_OR_SUPER_USER)]
+        [HttpPost("accounts/{accountname}/reprocess-transfer")]
+        public async Task<IActionResult> SendReprocessEmailForTransfer(string accountname, ReprocessByTransferUserData userData)
```
Comment (jhoffman-ms) [inline Doppler.BillingUser/Controllers/BillingController.cs:269]: Maybe the correct name is ```SendContactInformationForTransfer```. What do you think?

Comment (ms-fcabrera) [reply Doppler.BillingUser/Controllers/BillingController.cs:269]: Done!

File: Doppler.BillingUser/ExternalServices/EmailSender/EmailNotificationsConfiguration.cs [hunk 2]
```diff
@@ -29,5 +29,6 @@ public class EmailNotificationsConfiguration
         public Dictionary<string, string> UpdatePlanTemplateId { get; set; }
         public string UpdatePlanAdminTemplateId { get; set; }
         public string ReprocessStatusAdminTemplateId { get; set; }
+        public string ReprocessByTransferStatusAdminTemplateId { get; set; }
```
Comment (jhoffman-ms) [inline Doppler.BillingUser/ExternalServices/EmailSender/EmailNotificationsConfiguration.cs:32]: ```ContactInformationForTransferAdminTemplateId```. What do you think?

File: Doppler.BillingUser/Controllers/BillingController.cs [hunk 3]
```diff
@@ -264,6 +264,20 @@ public async Task<IActionResult> UpdateCurrentPaymentMethod(string accountname,
                 return new BadRequestObjectResult(e.Message);
             }
         }
+        [Authorize(Policies.PROVISORY_USER_OR_SUPER_USER)]
+        [HttpPost("accounts/{accountname}/reprocess-transfer")]
```
Comment (jhoffman-ms) [inline Doppler.BillingUser/Controllers/BillingController.cs:268]: ```send-contact-information-notification```. What do you think?

Comment (ms-fcabrera) [reply Doppler.BillingUser/Controllers/BillingController.cs:268]: Done!

**Pull request merged**